src/game-logic/reducers/makeRecipe.test.js

```diff
@@ -37,4 +39,34 @@ describe('makeRecipe', () => {
       ])
     })
   })
+
+  describe('experience', () => {
+    let state
+
+    beforeEach(() => {
+      state = {
+        experience: 0,
+        inventory: [],
+      }
+
+      jest.mock('../../utils', () => ({
+        ...jest.requireActual('../../utils'),
+        canMakeRecipe: jest.fn(() => true),
+      }))
```

Comment:
I don't think this mock is actually doing anything because it's being defined after `makeRecipe` is loaded. When I remove this mock, the tests still pass.

In order for this mock to actually be used, I think we'd need to reload `makeRecipe` with [`jest.requireActual`](https://jestjs.io/docs/jest-object#jestrequireactualmodulename). Here's an example of where that's done: https://github.com/jeremyckahn/farmhand/blob/9a8c5567f932e3acbf28342ec1defd58aa7b2dc2/src/utils/index.test.js#L801-L821

Reply:
good call. i wasn't thinking clearly when i did this haha